app/policies/organizations/adopter_application_policy.rb

```diff
@@ -0,0 +1,16 @@
+class Organizations::AdopterApplicationPolicy < ApplicationPolicy
```

Comment:
Should we name this policy AdopterApplicationReviewPolicy?

Reply:
`AdopterApplicationPolicy` makes more sense since it's still related to how applications are managed - https://github.com/rubyforgood/pet-rescue/pull/466#discussion_r1525688918

Reply:
Ok I was initially a little confused as we have a controller AdopterApplicationsController that enables an adopter to make an application then we have the AdopterApplicationReviews controller for reviewing said applications on the staff side. This current naming is not aligned in that sense?

Reply:
Ok I see there was discussion on this already.